### PR TITLE
Remove old grid from some components

### PR DESCRIPTION
**Pull request by maya**
[😎 Preview](https://federalist-proxy.app.cloud.gov/preview/uswds/uswds/remove-old-grid)

### CODE REVIEW

File: src/components/banner/banner.njk [hunk 1]
```diff
@@ -10,8 +10,8 @@
       </button>
       </div>
     </header>
-    <div class="usa-banner-content usa-grid usa-accordion-content" id="gov-banner">
-      <div class="usa-banner-guidance-gov usa-width-one-half">
+    <div class="usa-banner-content usa-accordion-content grid-container grid-row" id="gov-banner">
```
Comment (maya) [inline src/components/banner/banner.njk]: Would have liked to be able to add `grid-gap` but it messes up the margins - I'd prefer not to have to add another div.

Comment (maya) [reply src/components/banner/banner.njk]: cc @thisisdano

Comment (thisisdano) [reply src/components/banner/banner.njk]: Addressed in https://github.com/uswds/uswds/pull/2757/

File: src/components/layouts/landing-inner.njk [hunk 2]
```diff
@@ -15,8 +15,8 @@
   {% render '@graphic-list' %}
 
   <section class="usa-section">
-    <div class="usa-grid">
-      <h2>Section heading</h2>
+    <div class="grid-container">
```
Comment (thisisdano) [inline src/components/layouts/landing-inner.njk]: I feel like `usa-prose` _should_ work here, but it isn't. I'm not getting the `heading + *` spacing and the button text color is being overridden by the prose class...

screen shot 2018-09-20 at 10 21 19 am

I'll need to dig into this a bit more

File: src/components/layouts/landing-inner.njk [hunk 3]
```diff
@@ -15,8 +15,8 @@
...
+      <h2 class="font-heading-xl">Section heading</h2>
```
Comment (thisisdano) [inline src/components/layouts/landing-inner.njk]: This shouldn't be necessary if `usa-prose` were working properly

**Comment by maya** [comment]
@thisisdano FYI I reverted the banner changes in this PR in 741abbf since they're addressed in another PR.

File: src/components/layouts/landing-inner.njk (hunk 3 above)
Comment (maya) [reply src/components/layouts/landing-inner.njk]: Do we expect this section to be used beyond the current design (heading + lead + button), e.g. customized with additional text?

Comment (thisisdano) [reply src/components/layouts/landing-inner.njk]: Well, I guess I consider this section just an example of "something you could do". It's not really a component, per se, so it's not _really_ important whether it uses `usa-prose` or some other technique. It's certainly OK to leave as is.  

But it does reveal some of what I might consider bugs in `usa-prose`. We'll address them in a subsequent issue, so I'd say it's ok to leave this as-is and move this PR along.

File: src/components/layouts/landing-inner.njk (hunk 2 above)
Comment (thisisdano) [reply src/components/layouts/landing-inner.njk]: Just a note to self to address this in a subsequent issue.
https://github.com/uswds/uswds/issues/2758

**Pull request merged**